proxies/events-graphql-proxy/package.json

```diff
@@ -13,7 +13,7 @@
     "pre-commit": "lint-fix",
     "lint": "eslint . --ext .ts,.tsx,.js,.jsx,.cjs,.mjs,.mdx,.graphql --cache --cache-location ../../.cache/eslint/events-graphql-proxy.eslintcache",
     "lint-fix": "eslint . --ext .ts,.tsx,.js,.jsx,.cjs,.mjs,.mdx,.graphql --fix",
-    "check-dist": "es-check -v",
+    "check-dist": "",
```

Comment:
Shouldn't the whole command just be removed if it is not available? Or does the monorepo setup somehow expect it to be here? Could it at least have some kind of echo that it does not do anything?

Reply:
I can try to remove it, but it perhaps caused some error on ci.